varya/template-parts/post/author-bio.php

```diff
@@ -9,19 +9,21 @@
 
 if ( (bool) get_the_author_meta( 'description' ) ) : ?>
 <div class="author-bio">
+	<?php
+	printf(__( 'Published by', 'varya' ));
```

Comment:
Does this need to use `printf()` since it doesn't actually need a variable anymore? I think it could just use `_e()` as seen in the "View more posts" link down below.